core/src/main/java/feast/core/service/SpecService.java

```diff
@@ -275,7 +284,8 @@ public ListStoresResponse listStores(ListStoresRequest.Filter filter) {
 
   /**
    * Creates or updates a feature set in the repository. If there is a change in the feature set
-   * schema, then the feature set version will be incremented.
+   * schema, then the feature set version will be incremented. If there a project is not specified
```

Comment:
If the project is not specified in the given feature set, it will be assigned to the default project.

Reply:
Not sure why the comment is there. Currently, `listStore.Filter` has only one field: store name. Should we still be able to list stores by project? @woop

Reply:
No, sorry, i'm correcting the comment since it's a little hard to understand

Reply:
Corrected.

Reply:
Corrected.